fluent/ExtrinsicRune.ts

```diff
@@ -64,18 +64,29 @@ export class ExtrinsicRune<out C extends Chain, out U> extends PatternRune<Chain
       }))
   }
 
-  feeEstimate() {
+  weightRaw() {
     const arg = Rune
       .fn(concat)
       .call(this.extrinsic, this.extrinsic.access("length").map((n) => $.u32.encode(n)))
       .map(hex.encodePrefixed)
-    const data = this.chain.connection
+    return this.chain.connection
       .call("state_call", "TransactionPaymentApi_query_info", arg)
       .map(hex.decode)
+      .map((v) => v.slice(0, 6))
```

Comment:
The `TransactionPaymentApi_query_info` endpoint returns a heavily-padded hex str. If we don't strip the excess before passing to other calls, we get a runtime panic.

Reply:
Odd. Maybe we should add a comment in the code to prevent confusion in the future?

Can we be sure it is always just 7 bytes?

Reply:
> Can we be sure it is always just 7 bytes?

Should be consistent (as it's a u128 / 2^7)